rasa/core/trackers.py

```diff
@@ -131,10 +131,11 @@ def __init__(
         # id of the source of the messages
         self.sender_id = sender_id
         # slots that can be filled in this domain
-        if slots is not None:
-            self.slots = {slot.name: copy.deepcopy(slot) for slot in slots}
-        else:
-            self.slots = AnySlotDict()
+        self.slots = (
```

Comment:
OPTIONAL: I am not sure removing the if / else makes the code more readable, but I think either version works